.gitignore

```diff
@@ -4,6 +4,7 @@
 /app/server/public/css/
 /app/server/public/js/
 /app/server/public/hot/
+/app/server/public/fonts/
```

Comment:
Not sure this is needed, or why?

Reply:
this is just from the underlining code PR, for errors and warnings! I didnt put a git ignore for the font awesome fonts and realised it would end up being committed if i didnt add a git ignore for it